libs/barista-components/alert/src/alert.scss

```diff
@@ -43,8 +43,3 @@
   padding: 10px 16px;
   flex-grow: 1;
 }
-
```

Comment:
Here as well, can we move the import of the theme file down here, to keep the source order intact.